src/goto-harness/recursive_initialization.cpp

```diff
@@ -16,9 +16,63 @@ Author: Diffblue Ltd.
 #include <util/optional_utils.h>
 #include <util/std_code.h>
 #include <util/std_expr.h>
+#include <util/string2int.h>
```

Comment:
The commit message sounds cool, but really doesn't give my any information I'm afraid. Please clarify.

Reply:
Expanded here: https://github.com/diffblue/cbmc/pull/4482/commits/d96545dc3db54af5bed3d7d66dbc69efd48db6bd.